src/test/java/picard/sam/MergeBamAlignmentTest.java

```diff
@@ -954,7 +952,7 @@ public void testEarliestFragmentStrategyPaired() throws Exception {
         alignedSam.deleteOnExit();
 
         // Populate the header with SAMSequenceRecords
-        header.getSequenceDictionary().addSequence(new SAMSequenceRecord("chr1", 1000000));
+        header.setSequenceDictionary(SamReaderFactory.makeDefault().getFileHeader(sequenceDict2).getSequenceDictionary());
```

Comment:
Why not use the SAMSequenceDictionaryExtractor here and on the other similar lines?